### ISSUE TITLE
Value of the resistorpack is not on the PCB

### ISSUE THREAD

**Issue by JoopIdema** [misterblack1/isa-ram-expansion#5]
The value of the resistorpack is not marked on the PCB. I assume 4k7 or 10k would be ok?

**Comment by sfambach** [comment]
On the back site lower right corner is 4k7 mentioned if 74LS is used, nether the less a BOM would be good ;)

**Comment by JoopIdema** [comment]
No, that is the value for R2, not the value for the resistor-network RN1. I used 10K for RN1 and it worked fine.

**Comment by sfambach** [comment]
Ok I used 4k7 worked, too, but maybe it explaines some trouble I had. Some words from the master would be good ;)
